svg/elements/script.json

```diff
@@ -147,7 +147,8 @@
             "__compat": {
               "description": "<code>type='module'</code>",
               "tags": [
-                "web-features:svg2-script-html-equivalence"
```

Comment:
This is the only removal in this diff, but it's because of the comma. This might also be the only entry tagged with two features.